packages/tree-view/src/ux-tree-view/i-ux-tree-view-element.ts

```diff
@@ -0,0 +1,9 @@
+import { IController } from '@aurelia-ux/core';
+import { UxTreeView } from './ux-tree-view';
+
+export interface IUxTreeViewElement extends HTMLElement {
+  au: {
```

Comment:
Do we need this?

Reply:
Not at the moment, just for future needs